appveyor.yml

```diff
@@ -22,4 +22,13 @@ build_script:
 - ps: ./build.ps1
 test: off
 artifacts:
-- path: artifacts/packages/*.nupkg
\ No newline at end of file
+- path: artifacts/packages/*.nupkg
+deploy:
+- provider: NuGet
+  server: https://www.myget.org/F/skywalking/api/v2/package
```

Comment:
Why named under `v2`? Which version is this about?

Reply:
This is the push server, which you can see in myget's feed details